pkg/e2fsprogs

```diff
@@ -1,8 +1,10 @@
 [mirrors]
-http://prdownloads.sourceforge.net/e2fsprogs/e2fsprogs-1.42.9.tar.gz
+https://mirrors.edge.kernel.org/pub/linux/kernel/people/tytso/e2fsprogs/v1.42.9/e2fsprogs-1.42.9.tar.xz
```

Comment:
can you make that just kernel.org (like the now first url in the e2fs pkg)?
mirrors.edge is just a single one of the mirrors kernel.org redirects to